src/glossary.rst

```diff
@@ -92,10 +92,11 @@ Glossary
       :term:`Dependency <dependency>` relationships can be thought of the other
       way around as "triggers".
 
-      For example the dependency ``foo => bar`` could be described in two ways:
+      For example the dependency ``foo => bar`` could be described in several ways:
 
-      * "``bar`` is dependent on ``foo``"
-      * "``foo`` is triggered by ``bar``"
+      * "``bar`` depends on ``foo``"
+      * "``foo`` triggers ``bar``"
+      * "``bar`` triggers off of ``foo``"
```

Comment:
Using the American "off of" here because it is unambigous (The non-American "x triggers off y" implies the wrong direction to some people - like "x kicks off y").

Reply:
I agree that 'triggers off of' is less ambiguous than 'triggers off'. A bit of a mouthful but can't think of anything better if wanting to move away from 'is triggered by'.

Reply:
Well "`bar` is triggered by `foo`" is fine too, but the original line has it the wrong way around. 

Actually, "`bar` is triggered by `foo`" implies the upstream task `foo` is the protagonist,  whereas "`bar` triggers off of `foo`" suggests that the downstream task `bar` triggers itself in response to the state of `foo`.

Under spawn on submit, it is actually the latter case that is correct (to the extent that it matters) ... but in our spawn on demand future, the former case will be true.  

(Hopefully that's clear as mud!)